style.css

```diff
@@ -84,54 +84,54 @@ html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:1
 --------------------------------------------- */
 
 html,
-input[type="search"]{
+input[type="search"] {
 	-webkit-box-sizing: border-box;
-	-moz-box-sizing:    border-box;
-	box-sizing:         border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
 }
 
 *,
-*:before,
-*:after {
+*::before,
```

Comment:
First of all I hate IE8, second I like consistency and sometimes the script uses double colons and sometimes single. I prefer to use double and imo it is not possible to keep supporting very old browsers